frontend/src/views/ShootList.vue

```diff
@@ -16,6 +16,41 @@ SPDX-License-Identifier: Apache-2.0
           <div class="text-subtitle-1 toolbar-title--text">{{headlineSubtitle}}</div>
         </v-toolbar-title>
         <v-spacer></v-spacer>
+        <v-tooltip top>
+          <template v-slot:activator="{ on }">
+            <div v-on="on">
+              <v-badge
+                class="mr-5"
+                bordered
+                color="primary lighten-3"
+                :content="numberOfNewItemsSinceFreeze"
+                :value="numberOfNewItemsSinceFreeze > 0"
+                overlap
+              >
+                <v-switch
+                  v-model="freezeSorting"
+                  v-if="!projectScope && isAdmin"
+                  class="mr-3"
+                  color="primary lighten-3"
+                  hide-details>
+                  <template v-slot:label>
+                    <span class="text-subtitle-1 toolbar-title--text">Focus</span>
+                  </template>
+                </v-switch>
+              </v-badge>
+            </div>
+          </template>
+          <span class="font-weight-bold">Focus Mode</span><br />
+          Cluster List sorting is freezed.<br />
+          Items in the list will still be updated.<br />
+          New items will not be added to the list.<br />
+          Removed items will be shown as stale (greyed out).
+          <template v-if="numberOfNewItemsSinceFreeze > 0">
+            <v-divider color="white"></v-divider>
+            <span class="font-weight-bold">{{numberOfNewItemsSinceFreeze}}</span> new clusters were added to the list since you enabled focus mode.<br />
+            New clusters will not be added to the list until you disable focus mode.
```

Comment:
```suggestion
            New clusters with issues will not be added to the list until you disable focus mode.
```
also add to list items and show always